pkg/component/watch.go

```diff
@@ -50,26 +50,31 @@ func addRecursiveWatch(watcher *fsnotify.Watcher, path string, ignores []string)
 		}
 		return nil
 	})
+	fmt.Printf("\n\nIgnores is %s\n\n", ignores)
```

Comment:
Why there are two empty lines and beginning and end of this?
Shouldn't this be in debug log?